### PR TITLE
Issues/issue 5 addtional samplers

### PR DESCRIPTION
**Pull request by grovesy**

### CODE REVIEW

File: node_modules/nodemailer/.prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
```
Comment (zheyu-wang-tony) [inline node_modules/nodemailer/.prettierrc.js:1]: Should we add nodule_modules folder to git ignore?

**Comment by grovesy** [comment]
Sure will add


Sent with BlackBerry Work
(www.blackberry.com)

**Comment by grovesy** [comment]
node_modules removed

**Comment by zheyu-wang-tony** [comment]
@maoo Hi Mao, would you please help to check why PR need a pylint status check? I cannot disable it in the setting page since I don’t have enough entitlement.

**Comment by zheyu-wang-tony** [comment]
@grovesy Hi Paul, I think it’s not related to build action, we need to disable it in branch setting which is not visible to us right now.

**Comment by grovesy** [comment]
> @grovesy Hi Paul, I think it’s not related to build action, we need to disable it in branch setting which is not visible to us right now.

Yes noticed - thought I would just double check

**Comment by zheyu-wang-tony** [comment]
> @maoo Hi Mao, would you please help to check why PR need a pylint status check? I cannot disable it in the setting page since I don’t have enough entitlement.

@finos-admin

**Comment by finos-admin** [comment]
@zheyu-wang-tony - I've disabled `pylint` from the checks, I think it was an old check that was introduced by Probot Settings (now disabled).

The PR can be merged now. Sorry for the inconvenience

**Comment by grovesy** [comment]
wohoo .. cheers Mao

**Pull request merged**